force-app/main/default/lwc/lwcVoiceExtension/lwcVoiceExtension.html

```diff
@@ -0,0 +1,13 @@
+<!-- 
+    Copyright (c) 2021, salesforce.com, inc.
+    All rights reserved.
+    SPDX-License-Identifier: BSD-3-Clause
+    For full license text, see the LICENSE file in the repo root or https://opensource.org/licenses/BSD-3-Clause
+-->
+<template>
+    <lightning-service-cloud-voice-toolkit-api></lightning-service-cloud-voice-toolkit-api>
+    <lightning-card variant="Narrow" title="LWC Voice Extension">
```

Comment:
nit : Should we mention it is using voice tool kit API so that demo becomes more informative  ?